### PR TITLE
Dcadams/merge upstream commit 26c8622

### PR DESCRIPTION
**Pull request by dcadams**
Merge of upstream commit 26c8622.

This is required for the edx-platform merge of upstream branch hotfix-2015-11-10.

### CODE REVIEW

File: openassessment/workflow/serializers.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 class AssessmentWorkflowSerializer(serializers.ModelSerializer):
     score = serializers.ReadOnlyField(required=False)
-    override_score = serializers.Field(required=False)
+    override_score = serializers.ReadOnlyField(required=False)
```
Comment (stvstnfrd) [inline openassessment/workflow/serializers.py]: What's the reason for this?

Comment (dcadams) [inline openassessment/workflow/serializers.py]: This was due to their DjangoRestFramework upgrade. override_score is our field.

Comment (stvstnfrd) [inline openassessment/workflow/serializers.py]: Got it.

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**